app/merginapi.cpp

```diff
@@ -1101,6 +1118,37 @@ void MerginApi::pingMergin()
   connect( reply, &QNetworkReply::finished, this, &MerginApi::pingMerginReplyFinished );
 }
 
+void MerginApi::migrateProjectToMergin( const QString &projectName, const QString &projectNamespace )
+{
+  InputUtils::log( "migrate project", projectName );
+  if ( projectNamespace.isEmpty() )
+  {
+    createProject( mUserAuth->username(), projectName );
+  }
+  else
+  {
+    createProject( projectNamespace, projectName );
+  }
+}
+
+void MerginApi::detachProjectFromMergin( const QString &projectNamespace, const QString &projectName )
+{
+  // remove mergin folder
+  QString projectFullName = getFullProjectName( projectNamespace, projectName );
+  LocalProjectInfo projectInfo = mLocalProjects.projectFromMerginName( projectFullName );
+  if ( projectInfo.isValid() )
+  {
+    QDir merginProjectDir( projectInfo.projectDir + "/.mergin" );
+    merginProjectDir.removeRecursively();
+  }
+  // Update localProjects (updating mMerginProjects can be omitted since it is updated on listing projects)
+  mLocalProjects.resetMerginInfo( projectNamespace, projectName );
+  mLocalProjects.reloadProjectDir();
+
+  emit notify( QStringLiteral( "Project detached from Mergin" ) );
```

Comment:
Translate string `tr ()`

Reply:
Also messages from server (like `Project XYZ already exists!`) are probably not being translated but showed to user in english, maybe we could change that in future